sample/rules_test.go

```diff
@@ -2152,3 +2157,393 @@ func TestRulesWithDeterministicSampler(t *testing.T) {
 		assert.Len(t, spans, len(d.Spans), "should have the same number of spans as input")
 	}
 }
+
+func TestRulesRootSpanContext(t *testing.T) {
+	data := []TestRulesData{
+		//{
```

Comment:
Looks like you commented this out to have one test case but left it that way. Please fix.